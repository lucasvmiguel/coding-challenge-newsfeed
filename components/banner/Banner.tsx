import styled from 'styled-components'

type Props = {
  color: string
  bgColor: string
}

const Banner = styled.div<Props>`
  padding: 20px;
  text-align: center;
  background: ${props => props.bgColor};
  color: ${props => props.color};
  font-size: 14px;
`

export default Banner
