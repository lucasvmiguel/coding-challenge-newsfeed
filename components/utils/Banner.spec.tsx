import renderer from 'react-test-renderer'

import Banner from './Banner'

it('renders Banner component correctly - snapshot', () => {
  const tree = renderer
    .create(<Banner bgColor="black" color="white" />)
    .toJSON()
  expect(tree).toMatchSnapshot()
})
