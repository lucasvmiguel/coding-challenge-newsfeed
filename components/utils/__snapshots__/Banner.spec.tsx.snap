// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Banner component correctly - snapshot 1`] = `
<div
  className="sc-bdfBQB dwcRfT"
  color="white"
/>
`;
